Example/Example/AppDelegate.swift

```diff
@@ -10,22 +10,68 @@ import UserNotifications
 import RadarSDK
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate, CLLocationManagerDelegate, RadarDelegate, RadarVerifiedDelegate {
+class AppDelegate: UIResponder, UIApplicationDelegate, UIWindowSceneDelegate, UNUserNotificationCenterDelegate, CLLocationManagerDelegate, RadarDelegate, RadarVerifiedDelegate {
 
     let locationManager = CLLocationManager()
+    var window: UIWindow? // required for UIWindowSceneDelegate
+    
+    var scrollView: UIScrollView?
+    var demoFunctions = Array<() -> Void>()
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         UNUserNotificationCenter.current().requestAuthorization(options: [.alert, .sound]) { (_, _) in }
         UNUserNotificationCenter.current().delegate = self
-
+        
         locationManager.delegate = self
         self.requestLocationPermissions()
-
+        
         // Replace with a valid test publishable key
         Radar.initialize(publishableKey: "prj_test_pk_0000000000000000000000000000000000000000")
         Radar.setDelegate(self)
         Radar.setVerifiedDelegate(self)
+        
+        return true
+    }
+    
+    func createButton(text: String, function: @escaping () -> Void) {
+        guard let scrollView = self.scrollView else { return }
+        
+        let buttonHeight = 30
+        scrollView.contentSize.height += CGFloat(buttonHeight)
+        
+        let button = UIButton(frame: CGRect(x: 0, y: demoFunctions.count * buttonHeight, width: Int(scrollView.frame.width), height: buttonHeight),
+                              primaryAction:UIAction(handler:{ _ in
```

Comment:
is this formatting correct or just showing up funky on github?

Reply:
its a long line, I'll take out frame to its own variable